.gitlab-ci.yml

```diff
@@ -34,7 +31,7 @@ test:unit:
 build:make:
   stage: build
   needs: []
-  image: golang:1.17-alpine3.14
+  image: golang:1.21-alpine3.18
```

Comment:
Here we need to keep 1.17. See [this commit](https://github.com/mendersoftware/mender-connect/commit/1f09b9ab380019bc76711d214ec144b6c91e686d) for the reasoning.